components/pkg-export-docker/src/lib.rs

```diff
@@ -177,6 +175,7 @@ impl Credentials {
                 let auth_token_req = GetAuthorizationTokenRequest { registry_ids: None };
                 let token = client
                     .get_authorization_token(auth_token_req)
+                    .sync()
```

Comment:
For posterity, it should be noted that `sync` is a function [provided by Rusoto's Future implementation](https://rusoto.github.io/rusoto/rusoto_core/struct.RusotoFuture.html#method.sync), and not something that is generally available. But for not having to use the `futures` crate, it works like a charm.

Nice find!

Reply:
Good to know!